src/main/java/org/broadinstitute/hellbender/tools/exome/TargetArgumentCollection.java

```diff
@@ -31,7 +32,7 @@
  * </p>
  * <p>
  *   If even this mechanism does not provide a target file name, and the target file is deemed non-optional,
- *   the a {@link UserException.BadArgumentValue} is thrown.
+ *   the a {@link CommandLineException.BadArgumentValue} is thrown.
```

Comment:
I see -- but wasn't that true of `UserException` as well (which lives in gatk public, not protected)?